presto-main/src/main/java/com/facebook/presto/testing/LocalQueryRunner.java

```diff
@@ -529,6 +531,8 @@ public Lock getExclusiveLock()
             System.out.println(PlanPrinter.textLogicalPlan(plan.getRoot(), plan.getTypes(), metadata, session));
         }
 
+        planVerifiersFactory.verify(plan);
```

Comment:
It seems that `verify` is only called in `LocalQueryRunner`, which is only used in tests. It doesn't seem to be plugged into real query execution.